### ISSUE TITLE
Remove option at Push Publish Dialog not working

### ISSUE THREAD

**Issue by erickgonzalez** [dotCMS/core#19272]
If you select the option `Remove` at the Push Publish Dialog, nothing happens and an error at the log shows up.
```
15/09/20 16:41:00:246 GMT] ERROR ajax.RemotePublishAjaxAction: Unparseable date: "Invalid date-Invalid date"
java.text.ParseException: Unparseable date: "Invalid date-Invalid date"
at java.text.DateFormat.parse(DateFormat.java:366) ~[?:1.8.0_212]
at com.dotcms.publisher.ajax.RemotePublishAjaxAction.publish(RemotePublishAjaxAction.java:226) ~[dotcms_5.3.8_999999.jar:?]
```

This case is reproducible at demo (5.3.7) and 5.3.8

**Comment by erickgonzalez** [comment]
Tested Working as it should. Now able to do push using the Remove option without issues.

**Comment by bryanboza** [comment]
Fixed, tested on release-5.3.8 // Postgres // FF
